x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/no_entities_empty_state.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiCallOut,
+  EuiLink,
+  EuiEmptyPrompt,
+  EuiImage,
+  EuiHorizontalRule,
+  EuiText,
+  EuiTextColor,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+
+import { dashboardsLight } from '@kbn/shared-svg';
+import { useApmPluginContext } from '../../../../../context/apm_plugin/use_apm_plugin_context';
+import { useLocalStorage } from '../../../../../hooks/use_local_storage';
+import {
+  AddApmAgent,
+  AssociateServiceLogs,
+  CollectServiceLogs,
+} from '../../../../shared/add_data_buttons/buttons';
+
+export function NoEntitiesEmptyState() {
+  const { core } = useApmPluginContext();
+  const { basePath } = core.http;
+  const [userHasDismissedCallout, setUserHasDismissedCallout] = useLocalStorage(
+    'apm.uiNewExperienceCallout',
+    false
+  );
+
+  return (
+    <>
```

Comment:
This fragment could be removed

Reply:
https://github.com/elastic/kibana/pull/188209/commits/e8e93c72efd520a40a4541a9248dbc1669257ac3